MAUI/DataGrid/column-drag-and-drop.md

```diff
@@ -203,6 +203,47 @@ The SfDataGrid allows customizing the drag view text and background color using
 
 <img alt="DataGrid column drag view background and text color" src="Images\column-drag-and-drop\maui-datagrid-drag-view-background-and-text-color.png" width="450"/>
 
+### Column drag-and-drop template
+
+The SfDataGrid allows you to load specific content during column drag-and-drop operations using the [SfDataGrid.ColumnDragDropTemplate](). This template can be defined either in code or XAML.
```

Comment:
The `SfDataGrid` allows you to load specific content into the column drag-and-drop template using the `SfDataGrid.ColumnDragDropTemplate`. The content can be provided through either a `DataTemplate` or a `DataTemplateSelector`.

The following code snippet demonstrates how to load a data template into the column drag-and-drop view: